requirements.unittest.txt

```diff
@@ -0,0 +1,17 @@
+#Install the agent
+-e .
+
+# app
+git+https://github.com/openstack/neutron@9.1.0
+git+https://github.com/openstack/oslo.log.git@stable/liberty
+git+https://github.com/F5Networks/f5-openstack-lbaasv2-driver.git@liberty
+git+https://github.com/F5Networks/f5-openstack-test.git
+git+https://github.com/openstack/neutron-lbaas.git@stable/liberty
+
+f5-sdk==1.5.0
```

Comment:
Installing the agent should bring along the correct version of sdk, via setup.py.  This line won't work, in l7 for example, once the sdk slides forward.